### PR TITLE
Groovy Facade Utility Commands

### PR DESCRIPTION
**Pull request by MaxBorsch**
Ports the facade utility commands from gradle wrapper to groovy. 
Ex.)
`groovy facade` - For help
`groovyw facade get Server`
`groovyw facade create FacadeOtherthing`

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: facade.groovy [hunk 1]
```diff
@@ -0,0 +1,213 @@
+// We use GrGit for interacting with Git. This gets a hold of it as a dependency like Gradle would
+// TODO: Consider if we should do something to fix/suppress the SLF4J warning that gets logged on first usage?
+@GrabResolver(name = 'jcenter', root = 'http://jcenter.bintray.com/')
+@Grab(group = 'org.ajoberstar', module = 'grgit', version = '1.9.3')
+import org.ajoberstar.grgit.Grgit
+import org.ajoberstar.grgit.Remote
```
Comment (digitalripperynr) [inline facade.groovy]: Hey! I think this import is not used. Please consider removing this. There may be an increase in load times or just straight problems with this.

Comment (jellysnake) [reply facade.groovy]: To add onto this, the JsonSlurper remote also isn't used :)

**Comment by jellysnake** [comment]
Other than the imports it looks good. I'm gonna approve the task but wait for Cervator before merging because he knows groovy much better than I do and could point out any problems I've missed

**Comment by MaxBorsch** [comment]
@jellysnake Those unused imports are for the omitted auto-adding remotes to newly cloned facades. The module.groovy uses it but I didn't port over that part because it wasn't exactly part of the task. I left [bracketed] messages in the file showing where the auto-add remotes should go. I'm pretty sure all that needs to be done is copy the `addRemote()` from module.groovy to `facade.groovy` and then look at how `retrieveModule()` uses it and do the same for `retrieveFacade()`. Could be another simple coding task.

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by digitalripperynr** [comment]
#3206 aims to merge this functionality of this script and other groovyw scripts into a unified script. It contains common commands for modules, meta modules, facades and libs. It includes the '-remote' feature for all of them.

**Comment by Cervator** [comment]
Superseded / merged via #3263 - thanks! :-)

**Pull request closed without merging**